paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -179,6 +180,68 @@ class PaymentSheet internal constructor(
         IntentConfirmationInterceptor.createIntentCallback = createIntentCallback
     }
 
+    /**
+     * Builder to add optional callbacks to [PaymentSheet].
+     *
+     * @param resultCallback Called with the result of the payment after [PaymentSheet] is dismissed.
+     */
+    class Builder(internal val resultCallback: PaymentSheetResultCallback) {
+        internal var externalPaymentMethodConfirmHandler: ExternalPaymentMethodConfirmHandler? = null
```

Comment:
Do these need to be internal rather than private?

Reply:
They need to be `internal` so they are visible to `rememberPaymentSheet`